packages/diagrams/backend/sirius-components-diagrams-layout/src/test/java/org/eclipse/sirius/components/diagrams/layout/incremental/NodePositionProviderTests.java

```diff
@@ -61,7 +65,7 @@ public void testDiagramSeveralNewNodesAtOnce() {
         List<NodeLayoutData> nodes = new ArrayList<>();
         NodePositionProvider nodePositionProvider = new NodePositionProvider();
         DiagramLayoutData diagramLayoutData = this.createDiagramLayoutData();
-        NodeLayoutData nodeLayoutData = this.createNodeLayoutData(Position.UNDEFINED, DEFAULT_NODE_SIZE, diagramLayoutData, NodeType.NODE_RECTANGLE);
+        NodeLayoutData nodeLayoutData = this.createNodeLayoutData(Position.UNDEFINED, DEFAULT_NODE_SIZE, diagramLayoutData, NodeType.NODE_RECTANGLE, new FreeFormLayoutStrategy());
```

Comment:
Given that `new FreeFormLayoutStrategy()` is added to the majority of calls, maybe there should be an overload of `createNodeLayoutData()` which makes this the default.

Reply:
Done